Makefile

```diff
@@ -278,6 +278,12 @@ init-db-test-from-ls-test-server: db_ls_test_server_dump.trig init-db-test-empty
 	@curl -X POST -H "Content-Type: application/sparql-update" -d "DROP ALL" -u "admin:test" "http://localhost:3030/knora-test"
 	@curl -X POST -H "Content-Type: application/trig" --data-binary "@${CURRENT_DIR}/db_ls_test_server_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
 
+.PHONY: init-db-test-from-ls-test-server-trig-file
+init-db-test-from-ls-test-server-trig-file: init-db-test-empty ## init local database with data from a local ls-test-server dump
+	@echo $@
+	@curl -X POST -H "Content-Type: application/sparql-update" -d "DROP ALL" -u "admin:test" "http://localhost:3030/knora-test"
+	@curl -X POST -H "Content-Type: application/trig" --data-binary "@${CURRENT_DIR}/db_ls_test_server_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
+
```

Comment:
IMO we could have above `init-db-test-from-ls-test-server` make command split into 2 steps anyway
- dump the data first
- load it from that file

So basically command introduced by you stays and the other one could be reduced. Reason for that is the huge amount of data and the issue with that file if the download dies before it ends.

Reply:
That was my initial thought. (Additionally, once downloaded, you don't need the password anymore, which is nice.)

But I didn't want to mess with the other make target, so I thought I'd add a separate one. Do you think it makes a difference?

Reply:
Could you quickly add what Marcin suggested? It follows the pattern we are using in other cases.